docs/analytics.md

```diff
@@ -0,0 +1,22 @@
+# Flank Analytics
+
+Flank makes use of Bugsnag as its only analytics platform. It uses Bugsnag as an error monitor and project stability tool.
```

Comment:
maybe instead of analytics, it's more accurate to rebrand this as Error Monitoring

> Automated error monitoring, reporting, alerting
>
> https://www.bugsnag.com/

bugsnag tracks errors in flank, it doesn't watch what users do.

Reply:
Sure reworded especially as analytics is ambiguous here.